recipe/conda_build_config.yaml

```diff
@@ -1,5 +1,5 @@
 MACOSX_SDK_VERSION:  # [osx and x86_64]
-  - 10.12            # [osx and x86_64]
+  - 10.13            # [osx and x86_64]
 occt:
-  - 7.6.3
-  - 7.7.2
\ No newline at end of file
+  - 7.7.2
+  - 7.8.1
```

Comment:
This typically makes migrations much harder. Do you have a real need for 2 version of OCCT for one of your packages? Or are you trying to be accommodating in a hypothetical case?

Reply:
For example, the migration https://github.com/conda-forge/gmsh-feedstock/pull/90 failed because of these pins.

Reply:
Current development builds of freecad are still using occt 7.7.2. I guess that's the reason for having occt7.7.2 added here.